docs/UserGuide.md

```diff
@@ -49,10 +49,10 @@ If you can type fast, Tutor's pet can get your class management tasks done faste
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [tag/TAG]` can be used as `n/John Doe tag/student` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/Experienced`, `t/Average t/Potential TA` etc.
+* Items with `…​` after them can be used multiple times including zero times.<br>
+  e.g. `[tag/TAG]…​` can be used as ` ` (i.e. 0 times), `tag/student`, `tag/TA` etc.
```

Comment:
The second example should include more than one `tag` to show the possible multiplicities.
```suggestion
  e.g. `[tag/TAG]…​` can be used as ` ` (i.e. 0 times), `tag/student`, `tag/Average tag/TA Candidate` etc.
```

Reply:
Added the tags, used tag/average and tag/TA candidate to keep formatting consistent